packages/wp-now/src/wp-now.ts

```diff
@@ -278,6 +293,22 @@ async function initWordPress(
 	php.setPhpIniEntry('auto_prepend_file', configFile);
 }
 
+async function activatePluginOrTheme(
+	php: NodePHP,
+	{ projectPath, mode }: WPNowOptions
+) {
+	if (mode === WPNowMode.PLUGIN) {
+		const pluginFile = getPluginFile(projectPath);
+		console.log('pluginFile', pluginFile);
```

Comment:
Unexpected `console.log()` here.

However... it would be nice to output when this behavior is happening.

Reply:
I removed the log:
https://github.com/WordPress/wordpress-playground/pull/351/commits/28b2bac9937e6017759e99e61a09db749dada988